### DISCUSSION TITLE
New to Home assistant and HACS

### DISCUSSION THREAD

**Discussion by appollomm** [sebr/bhyve-home-assistant#52] [Q&A] [answered]
Hi I'm new to Home automation and i think i may of picked a tricker one to cut my teeth on but i currently have a bhyve tap controller and is my first project to see how this will work before i start investing in all the other cool automations that i want to implement :) 

I have Home Assistant and HACS installed, I got the bhyve home assistant package installed, but I'm not sure what to do with it from there, when I go into HACS I see the integration but how do i use it from there and obviously there are some customisation required to set the login and password etc.

if anyone is able to point me in the right direction it would be greatly appreciated

kind regards
Andrew

**Comment by sebr** [comment, marked as answer]
Hi @appollomm
You'll need to configure this integration via Home Assistant's `configuration.yaml` file which exists in the root of the HA installation folder. Once you've added the configuration as described in the project's README, please restart HA and the entities should become available.

There is no support for the UI wizard configuration at this stage.

Thanks and hth

**Reply by appollomm** [reply]
thanks watched a few vids, read some docs and have it up and running, love your work 👍
